pkg/tenant/api/v1/client_test.go

```diff
@@ -277,6 +277,33 @@ func TestVerifyEmail(t *testing.T) {
 	require.NoError(t, err, "could not execute verify request")
 }
 
+func TestResendEmail(t *testing.T) {
+	fixture := &api.Reply{}
+
+	// Create a test server
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		require.Equal(t, http.MethodPost, r.Method)
+		require.Equal(t, "/v1/resend", r.URL.Path)
+
+		w.Header().Add("Content-Type", "application/json; charset=utf-8")
+		w.WriteHeader(http.StatusOK)
+		json.NewEncoder(w).Encode(fixture)
+	}))
```

Comment:
Might be good to port the serve fixture helper from Quarterdeck to Tenant to reduce code in the future.